core/src/test/scala/org/apache/spark/PartitioningSuite.scala

```diff
@@ -244,6 +244,10 @@ class PartitioningSuite extends SparkFunSuite with SharedSparkContext with Priva
     assert(abs(6.0/2 - rdd.mean) < 0.01)
     assert(abs(1.0 - rdd.variance) < 0.01)
     assert(abs(1.0 - rdd.stdev) < 0.01)
+    assert(abs(rdd.variance - rdd.popVariance) < 1e-14)
+    assert(abs(rdd.stdev - rdd.popStdev) < 1e-14)
+    assert(abs(2.0 - rdd.sampleVariance) < 0.01)
+    assert(abs(1.41 - rdd.sampleStdev) < 0.01)
```

Comment:
Right. I'll fix that like `assert(abs(Math.sqrt(2.0) - rdd.sampleStdev) < 1e-14)`, too.